src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
I would write types for shapes and colors. So that no additional data can be transmitted.

Reply:
even not types, but enums 🤔